src/Language/PureScript/Sugar/ObjectWildcards.hs

```diff
@@ -137,7 +137,7 @@ desugarDecl other = fn other
         goLayer path (key, Branch branch) =
           let path' = path ++ [key]
               updates = goLayer path' <$> M.toList branch
-              accessor = foldr Accessor val path'
+              accessor = foldl (flip Accessor) val path'
```

Comment:
it's just a little thing, but `foldl'` is preferrable to `foldl` in almost all cases